api/handlers/sendgrid.js

```diff
@@ -0,0 +1,41 @@
+var sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
```

Comment:
Please import from `/config/credentials` as is done in the `airtable` handler instead of pulling directly from `process.env`